test/ts-types/throws.ts

```diff
@@ -0,0 +1,26 @@
+import test from '../..';
+
+class CustomError extends Error {
+	foo: string;
+
+	constructor() {
+		super();
+		this.foo = 'foo';
+	}
+}
+
+test('throws', t => {
+	const err1: Error = t.throws(() => {});
+	// t.is(err1.foo, 'foo');
```

Comment:
?

Reply:
It's a type exception. I think maybe there's a way to indicate that to `tsc`?

Reply:
You can use the `// @ts-ignore` comment to ignore the below line.